src/screens/AccountRecover.js

```diff
@@ -98,11 +144,11 @@ class AccountRecoverView extends React.Component {
           <Background />
           <Text style={styles.titleTop}>RECOVER ACCOUNT</Text>
           <Text style={styles.title}>CHOOSE NETWORK</Text>
-          <NetworkButton network={network} />
+          <NetworkButton network={selectedNetwork} />
```

Comment:
small thing, but i believe now we should change the order of these networks, i.e. Kusama, Polkadot at the top, and the rest as is.

Reply:
This is actually linked to the networkKey, ethereuem with 1, 2.. is getting first. There will be only Kusama for v3 and it'll be the default, so I guess it's ok.